### PR TITLE
Add openseadragon

### PR DESCRIPTION
**Pull request by jonathangreen**
## Ticket 
https://github.com/Islandora-CLAW/CLAW/issues/693

(I edited the wrong ticket - Nat)!

### CODE REVIEW

**Comment by whikloj** [comment]
👍  `vagrant up`

File: inventory/vagrant/group_vars/webserver/drupal.yml [hunk 1]
```diff
@@ -41,3 +41,8 @@ drupal_trusted_hosts_file: "{{ drupal_core_path }}/sites/default/settings.php"
 drupal_services_file: "{{ drupal_core_path }}/sites/default/services.yml"
 drupal_public_filesystem: "{{ drupal_core_path }}/sites/default/files"
 drupal_public_filesystem_user: www-data
+
+drupal_config_override:
+  - configname: openseadragon.settings
+    key: iiif_server
+    value: "'http://localhost:8080/cantaloupe'"
```
[the remark this answers is not in the document]
Comment (whikloj) [reply inventory/vagrant/group_vars/webserver/drupal.yml]: But thanks as this seems to have revealed that you can't (or I can't) edit and save the IIIF Server URL. The other settings work, but this one seems to remain the same always I'll have to look into it.

**Comment by jonathangreen** [comment]
Fixed that URL. Probably the reason it can't be changed is the way I'm setting the URL in the settings.php file. Maybe we just want to set it with drush instead? Its much easier to automate but putting into the settings.php file.

**Comment by whikloj** [comment]
It looks like it **is** being changed in the actual config but the changed value is not being picked up so maybe you are right? Perhaps we should template a openseadragon.settings config?

**Comment by jonathangreen** [comment]
Kind of looks like overrides work differently with the new configuration system: 
https://www.drupal.org/docs/8/api/configuration-api/configuration-override-system

What do you think? Should I change it to use brush to set the variable?

**Comment by whikloj** [comment]
Yeah I think we may need to.

**Comment by jonathangreen** [comment]
@whikloj I changed how I set the variable. Its kind of ugly, but its idempotent.

**Comment by whikloj** [comment]
Cool, I'll test it tomorrow. That is a bit of a mess, but I don't have any better ideas 😄

**Comment by whikloj** [comment]
Looks good, we still have to manually set the field formatter. Not sure if we could automate that as a separate ansible task? If so, that would be awesome.

👍  merging

**Pull request merged**